static_src/actions/error_actions.js

```diff
@@ -38,10 +38,10 @@ export default {
   },
 
   importantDataFetchError(err, entityMessage) {
-    let msg = 'Page failed to load, please try again';
+    let msg = 'Connection issue, please try again';
```

Comment:
I like that, but from the user testing, it seems like the specific messaging was useful, so I want to  keep the "app usage data might not be available" in there.